src/RefractAPI.cc

```diff
@@ -39,6 +41,18 @@ namespace drafter {
             }
         }
 
+        refract::ArrayElement* CreateArrayElement(refract::IElement* value, bool rFull = false)
```

Comment:
I don't understand to context. It probably comes while merge. I do not use "CreateArrayElement" anewhere.